### PR TITLE
Setup Repository README to Document Project Setup

### PR DESCRIPTION
**Pull request by feng-eric**
### Notion ticket link

[Update Project README](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7&p=e45c94363bd5458ab6a27fde351980a5)

### Implementation description

- Set up the README for the repository
- Added in sample environment variable file

### Steps to test

1. Read over the README and see if there's anything else missing that should be added.

### What should reviewers focus on?

-

### Checklist

- [ ] My PR name is descriptive and in imperative tense
- [ ] I have run the linter
- [ ] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+SAMPLE_VARIABLE=
```
Comment (mahadzaryab1) [inline .env.sample:1]: should this file be committed?

File: README.md [hunk 2]
```diff
@@ -1,138 +1,100 @@
...
 
-## Getting Started
+This project was generated with [superplate](https://github.com/pankod/superplate).
 
-superplate is a Next.js all-in-one project generator. Create your project with the tools you need without spending hours on setting them up.
+## Team
+Project Lead: Eric Feng
+Product Manager: Raewyn Tsai
+Designers: Stacy Kwok, Christina Ru
+Developers: Jason Huang, Soha Khan, Cindy Wang, Brandon Wong, Victor Yun, Mahad Zaryab
```
Comment (mahadzaryab1) [inline README.md]: nit: consider adding these as bullet points so they render as a list - they're currently just being rendered together in one line

File: README.md [hunk 3]
```diff
@@ -1,138 +1,100 @@
...
 
-Every plugin comes with an example to give you a brief knowledge about their usage.
+## General Architecture
 
-## Available Scripts
+1. [NodeJS](https://nodejs.org/en/) application powered by the [Next.JS](https://nextjs.org/)
+   framework.
```
Comment (mahadzaryab1) [inline README.md]: nit: some points end in periods and some do not - consider standardizing it?

File: README.md [hunk 4]
```diff
@@ -1,138 +1,100 @@
...
 
-superplate is a Next.js all-in-one project generator. Create your project with the tools you need without spending hours on setting them up.
+## Team
+Project Lead: Eric Feng
+Product Manager: Raewyn Tsai
+Designers: Stacy Kwok, Christina Ru
+Developers: Jason Huang, Soha Khan, Cindy Wang, Brandon Wong, Victor Yun, Mahad Zaryab
 
-Every plugin comes with an example to give you a brief knowledge about their usage.
+## General Architecture
```
Comment (mahadzaryab1) [inline README.md]: nit: what about calling this **Project Architecture** - seems more specific?

File: README.md [hunk 5]
```diff
@@ -1,138 +1,100 @@
...
-
-[Go To Documentation](https://testing-library.com/docs/)
-
-### **Cypress**
-
-Fast, easy and reliable testing for anything that runs in a browser.
+To run the linters manually:
+```bash
+# Runs linting
+yarn linting
```
Comment (mahadzaryab1) [inline README.md]: I believe this should be `yarn lint`. I get an error if I run `yarn linting`.

File: README.md [hunk 6]
```diff
@@ -1,138 +1,100 @@
-# social-diversity-for-children
+# Social Diversity for Children Foundation
 
-This project was generated with [superplate](https://github.com/pankod/superplate).
+Program registration platform for [Social Diversity for Children Foundation](https://www.socialdiversity.org).
```
Comment (mahadzaryab1) [inline README.md]: I think it would be nice to get a bit more detail here - what exactly is it that we're building? It would also be good to mention it's currently in development.

File: .gitignore [hunk 7]
```diff
@@ -5,6 +5,6 @@ node_modules
 # environment variables
 .env
 .env.*
-!.env.example
+!.env.sample
```
Comment (mahadzaryab1) [inline .gitignore:8]: why is this required if the line above has `.env.*`?

**Review by mahadzaryab1 — CHANGES_REQUESTED** [review]
Added a few small comments. Some other things to consider: 
-  I believe we should also look at adding some details surrounding running the tests. 
- nit: The PR title should be in the imperative - maybe something like **Setup Repository README to Document Project Setup**

File: .env.sample (hunk 1 above)
Comment (feng-eric) [reply .env.sample:1]: Yes, it's a sample env file that will have the variables but not the values

File: README.md (hunk 5 above)
Comment (feng-eric) [reply README.md]: Good catch

File: .gitignore (hunk 7 above)
Comment (feng-eric) [reply .gitignore:8]: `!` prefix negates the pattern so it makes sure that `.env.sample` can be included

File: README.md (hunk 6 above)
Comment (feng-eric) [reply README.md]: Added a TODO

File: README.md (hunk 2 above)
Comment (feng-eric) [reply README.md]: Added new line

File: README.md [hunk 8]
```diff
@@ -1,21 +1,23 @@
 # Social Diversity for Children Foundation
 
-Program registration platform for [Social Diversity for Children Foundation](https://www.socialdiversity.org).
+Program registration platform for [Social Diversity for Children Foundation](https://www.socialdiversity.org). 
+
+Currently in development!
 
 This project was generated with [superplate](https://github.com/pankod/superplate).
```
Comment (mahadzaryab1) [inline README.md:7]: do we need this?

**Review by mahadzaryab1 — APPROVED** [review]
LGTM - just one small comment.

File: README.md (hunk 8 above)
Comment (feng-eric) [reply README.md:7]: Don't think it matters or is bad to have
